configs/BAYCK_DOLPHIN_F435/config.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     AT32F435G
+
+#define BOARD_NAME        BAYCK DOLPHIN
+#define MANUFACTURER_ID   BAYCKRC
```

Comment:
manufacturer id is 4 characters only.

Reply:
Please mention website url for the product.

Reply:
We currently have a website, but we haven't had time to build it yet. It is expected to be completed within 3 months. Also, does the website need to be written to the config. h file?

Reply:
No there is no need to include url in the config. We need to have some info on the board itself. 
We have a wiki link for each board pointing to documentation (like board specification, layout and wire diagram)